samples/auth-sample/auth-consumer/pom.xml

```diff
@@ -29,6 +29,12 @@
     <dependency>
       <groupId>org.apache.servicecomb</groupId>
       <artifactId>transport-highway</artifactId>
+      <exclusions>
```

Comment:
is this necessary?

Reply:
of course. If do not  have this ...
the log is as bellow.
```
log4j:WARN No appenders could be found for logger (org.springframework.core.env.StandardEnvironment).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```